pkg/kubelet/config/file_test.go

```diff
@@ -36,7 +38,7 @@ import (
 func TestExtractFromNonExistentFile(t *testing.T) {
 	ch := make(chan interface{}, 1)
 	c := sourceFile{"/some/fake/file", "localhost", ch}
-	err := c.extractFromPath()
+	err := c.watch()
```

Comment:
Hi @MrHohn 
`watch()` is the entrance and it will call `extractFromPath()`.  So I want to make sure this feature is still work from end to end.